### PR TITLE
Add /get_project by project_id endpoint

### PR DESCRIPTION
**Pull request by elsiel23**

### CODE REVIEW

File: src/planscape/plan/tests.py [hunk 1]
```diff
@@ -439,8 +442,49 @@ def test_good_polygon(self):
...
+            {'project_id': self.project_no_user.pk, 'geometry': {'features': [
                 {'geometry': {'type': 'MultiPolygon', 'coordinates': [[[[1, 2], [2, 3], [3, 4], [1, 2]]]]}}]}},
             content_type='application/json')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(ProjectArea.objects.count(), 1)
+
+
+class GetProjectTest(TransactionTestCase):
+    def setUp(self):
+        self.user = User.objects.create(username='testuser')
```
Comment (riecke) [inline src/planscape/plan/tests.py:454]: Since there is a "user", can you add a few tests to check when the user is logged in?

Comment (elsiel23) [reply src/planscape/plan/tests.py:454]: Added mirroring tests for a signed in user, let me know if you think we could use more

**Review by riecke — APPROVED** [review]
Nice tests.  I feel like Columbo, "Just one more thing..." Could we have a failure test, where the id doesn't match anything in the database?  Other than that, LGTM.

**Review by strangeties — APPROVED** [review]

**Comment by elsiel23** [comment]
@riecke Added!

**Pull request merged**